packages/flutter/lib/src/material/material_localizations.dart

```diff
@@ -6,6 +6,7 @@ import 'dart:async';
 
 import 'package:flutter/foundation.dart';
 import 'package:flutter/widgets.dart';
+import 'package:intl/intl.dart';
```

Comment:
please don't import package:intl into package:flutter.

Reply:
Filed https://github.com/flutter/flutter/issues/12050 on this.

Reply:
For now MaterialLocalizations does depend on package:intl. Once the date/time picker changes have landed, and we've sorted out how packages can provide their own localizations, I'll migrate this version of MaterialLocalizations to a separate opt-in package and replace it with an english-only stub.